interactions/client/models/command.py

```diff
@@ -436,7 +435,6 @@ class Command(DictSerializerMixin):
     )
     recent_group: Optional[str] = field(default=None, init=False)
     error_callback: Optional[Callable[..., Awaitable]] = field(default=None, init=False)
-    resolved: bool = field(default=False, init=False)
```

Comment:
state a reason for the removal of this attribute

Reply:
When a command is created in an extension, only a single Command instance is made. Having a single attribute to show it the command is resolved prevents multiple clients from losing that command